js/src/client.ts

```diff
@@ -3832,6 +3834,75 @@ export class Client {
     );
   }
 
+  /**
+   * Upsert examples with attachments using multipart form data.
+   * @param upserts List of ExampleUpsertWithAttachments objects to upsert
+   * @returns Promise with the upsert response
+   */
+  public async upsertExamplesMultipart({
```

Comment:
why not just make the input type `ExampleUpsertWithAttachments[]` or `Array<ExampleUpsertWithAttachments>`?